browser/ui/webui/brave_vpn/panel_handler/BUILD.gn

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+source_set("panel_handler") {
```

Comment:
I think we don't need to have separate target for `panel_handler`.
I think all these can be added here - https://github.com/brave/brave-core/pull/9730/files#diff-7ce4cface9f839816f0fc02bad0e04aa80284b048ef0d8f0876547e6e37e38b6R478

Reply:
IMO, panel_handler impls could be in `brave_vpn` directory together with others.
but it's up to you.